Mathlib/Logic/Embedding/Basic.lean

```diff
@@ -114,6 +114,7 @@ theorem coeFn_mk {α β} (f : α → β) (i) : (@mk _ _ f i : α → β) = f :=
 theorem mk_coe {α β : Type*} (f : α ↪ β) (inj) : (⟨f, inj⟩ : α ↪ β) = f :=
   rfl
 
+@[grind! .] -- This adds `Injective f` into the grind context for every embedding `f : α ↪ β`.
```

Comment:
When used as a focusing dot, mathlib prefers `·` over `.`; do we want the same for grind annotations?